txmongo/protocol.py

```diff
@@ -404,6 +404,10 @@ def opcode(self):
         #        Maybe we should refactor other messages too. Or get rid of MsgHeader dataclass.
         return self.header.opcode
 
+    @classmethod
+    def create_flag_bits(cls, not_more_to_come: bool ) -> int:
```

Comment:
How about `black` in `pre-commit`? :)

```suggestion
    def create_flag_bits(cls, not_more_to_come: bool) -> int:
```

Reply:
need install pre-commit on every machine!)